cmd/root.go

```diff
@@ -9,17 +9,21 @@ import (
 	"github.com/matt-simons/ss/pkg"
 	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
```

Comment:
@danielerez  can we use "sigs.k8s.io/yaml" as it is already used in pkg/ss.go?

Reply:
done